### PR TITLE
Basic False Negatives

### PR DESCRIPTION
**Pull request by haroldhues**
With much trial and error, SemDiff will now be able to detect the false positive set up in Curly-Broccoli!

Also have a test that insures that the functionality works, at least in the basic case!

### CODE REVIEW

**Comment by haroldhues** [comment]
Close #4

**Comment by haroldhues** [comment]
Close #5

**Comment by cartermp** [comment]
> SemDiff will now be able to detect the false positive set up in Curly-Broccoli!

:+1:

File: SemDiff.Core/Repo.cs [hunk 1]
```diff
@@ -103,8 +106,13 @@ internal Repo(string directory, string owner, string name)
         {
             if (Authentication)
             {
-                var authToken = gitHubConfig.AuthenicationToken;
+#if DEBUG
+                var authUsername = "haroldhues";
+                var authToken = "9db4f2de497905dc5a5b2c597869a55a9ae05d9b";
```
Comment (haroldhues) [inline SemDiff.Core/Repo.cs]: :(

**Pull request merged**